generators/client/templates/angular/src/main/webapp/app/polyfills.ts.ejs

```diff
@@ -18,4 +18,5 @@
 -%>
 import 'core-js/proposals/reflect-metadata';
 import 'zone.js/dist/zone';
+import '@angular/localize/init';
```

Comment:
as per above comments, check if required

Reply:
Yes it is, without invoking init() method, the app cannot start